Sources/XcodeGraphMapper/Mappers/Targets/PBXTargetMapper.swift

```diff
@@ -229,19 +222,14 @@ struct PBXTargetMapper: PBXTargetMapping {
             headers: headers,
             coreDataModels: coreDataModels,
             scripts: scripts,
-            environmentVariables: environmentVariables,
-            launchArguments: launchArguments,
             filesGroup: filesGroup,
             dependencies: allDependencies,
             rawScriptBuildPhases: rawScriptBuildPhases,
             playgrounds: playgrounds,
             additionalFiles: additionalFiles,
             buildRules: buildRules,
-            prune: prune,
             mergedBinaryType: mergedBinaryType,
-            mergeable: mergeable,
             onDemandResourcesTags: onDemandResourcesTags,
-            metadata: metadata,
```

Comment:
The items removed here do not map directly to properties in an XcodeProj file.  Most of these parameters control Tuist specific behaviors like scheme generation or adding build settings.  Since this mapper is reading already generated `.pbxproj` files the values there should be the source of truth and we should not reconstruct Tuist specific information from build settings.